encoding/protobuf/testing/benchmark_test.go

```diff
@@ -66,7 +67,7 @@ func BenchmarkIntegrationGRPCAll(b *testing.B) {
 	}
 	go func() { _ = server.Serve(listener) }()
 	defer server.Stop()
-	grpcClientConn, err := grpc.Dial("0.0.0.0:1234", grpc.WithInsecure())
+	grpcClientConn, err := grpc.Dial("0.0.0.0:1234", grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
grpc.WithInsecure became deprecated, and linter insists on changing the call.

It's straightforward replacement: grpc.WithInsecure sets insecure.NewCredentials() to transport creds, so we just making it explicitly now.